quickstarts/install-netobserv.yaml

```diff
@@ -0,0 +1,62 @@
+  tags:
+    - network
+    - observability
+    - netobserv
+  tasks:
+    - description: |-
+        You can install the Network Observability Operator using the OpenShift Container Platform web console Operator Hub. When you install the Operator, it provides the FlowCollector custom resource definition (CRD). You can set specifications in the web console when you create the FlowCollector.
+        1. In the OpenShift Container Platform web console, click **OperatorHub** from the [Operators]{{highlight qs-nav-operators}} → OperatorHub.
+        2. Choose **Network Observability Operator** from the list of available Operators in the OperatorHub, and click **Install**.
+        3. Select the checkbox **Enable Operator recommended cluster monitoring on this Namespace**.
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: |-
+          #### Verify the operator was successfully installed:
+          Navigate to Operators → Installed Operators.
+          Is Network Observability Operator listed ?
+      summary:
+        failed: Try the steps again.
+        success: Great work! You successfully installed Network Observability Operator.
+      title: Install the Network Observability Operator
+    - description: |-
+        In order to start capturing traffic, you need to create a **FlowCollector** resource to configure the components of Network Observability Operator.
+
+        1. Navigate to Operators → Installed Operators and select **Network Observability Operator**.
+
+        2. Navigate to the **FlowCollector** tab, and click **Create FlowCollector**. Make the following selections in the form view:
+          
+          2.1 **spec.agent.ebpf.Sampling**: Specify a sampling size for flows. Lower sampling sizes will have higher impact on resource utilization. For more information, see the "FlowCollector API reference", spec.agent.ebpf.
+          
+          2.2 If you are not using Loki, click Loki client settings and change **Enable** to **False**. The setting is True by default.
+          
+          2.3 If you are using Loki, set the following specifications:
+          
+            i. **spec.loki.mode**: Set this to the **LokiStack** mode, which automatically sets URLs, TLS, cluster roles and a cluster role binding, as well as the authToken value. Alternatively, the Manual mode allows more control over configuration of these settings.
+          
+            ii. **spec.loki.lokistack.name**: Set this to the name of your LokiStack resource. In this documentation, loki is used.
```

Comment:
@stleerh you mentionned you would like to point out Loki Operator doc. How do you want to handle it here ?